docs/reference_guide.md

```diff
@@ -781,6 +783,8 @@ Loops can be short circuited by using the `continue` and `break` keywords.
 The `return` keyword is used to exit the current probe. This differs from
 `exit()` in that it doesn't exit bpftrace.
 
+By default zero value is returned, but user can specify expression to be returned.
```

Comment:
I think it would be good to provide a bit more detail here as this might give users the impression that the return value has a meaning. Atm it is only relevant for the LSM and for the few kprobes that support it `override()` can be used

Reply:
ok, will add